apps/gauzy/src/app/pages/settings/oauth-clients/oauth-client-form-dialog/oauth-client-form-dialog.component.html

```
@@ -0,0 +1,138 @@
+		{{
+			(isEdit ? 'OAUTH_CLIENTS.DIALOG.EDIT_TITLE' : 'OAUTH_CLIENTS.DIALOG.CREATE_TITLE')
+				| translate
+		}}
+	</nb-card-header>
+
+	<nb-card-body>
+		<form [formGroup]="form" (ngSubmit)="save()">
+			<div class="form-row">
+				<label class="label">{{ 'OAUTH_CLIENTS.FORM.NAME' | translate }}</label>
+				<input
+					nbInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)